plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/ui/AppEngineStandardProjectConvertCommandHandlerTest.java

```diff
@@ -113,6 +115,8 @@ public void testCheckFacetCompatibility_java1_6FacetIsIncompatible() throws Core
 
   @Test
   public void testCheckFacetCompatibility_java1_8FacetIsIncompatible() throws CoreException {
+    Assume.assumeTrue("AppEngine Standard Java 8 runtime support is not present",
```

Comment:
so what is this testing? that the app engine facet isn't compatible with java 8 facet, but only sometimes? This feels off to me. Why should we not simply make the app engine standard facet unconditionally compatible with Java 8?